### ISSUE TITLE
Create a visualisation of cultural organisations across Leeds

### ISSUE THREAD

**Issue by gilesdring** [open-innovations/leeds-schools-cultural-engagement#20]
Sources - the union of

* List of orgs mentioned in Survey
* Orgs funded by ACE
* Orgs funded by LCC
* Orgs pulled from OSM (separate ticket required???)

**Comment by gilesdring** [comment]
What data to show for each?

**Comment by taz-chiles** [comment]
This is currently blocked on agreeing which data we want to display. Meeting with Bryony to clarify some questions around data quality. 

We may wish to show: 

Headline stats: 
- Number of cultural organisations funded by LCC
- Number of NPOs in Leeds
- Number of Arts Award centres in Leeds
- Number of NPOs which are Arts Award centres. 

Might we also want to show the number of Arts Council funded projects?

Map: 
3 different coloured markers, highlighting: 
- NPOs
- LCC funded organisations
- Other organisations that were mentioned in the survey. 

Tooltip to include (lower priority) 
- Does this organisation offer an Arts Award? Y/N?
- Link to their website

Other potential visualisations: 
- A table to rank NPOs by number of engagements with schools 
- A table to rank NPOs by amount of funding from ACE
- A table to rank organisations by funding from LCC
- A table or headline figure for Music Education Partnerships (more data on this to follow from LCC)
